src/Generator/Program.fs

```diff
@@ -213,15 +223,17 @@ let getEnumCasesAndCommandsPerVersion (data: OpenGL_Specification.Registry) =
     aggregateFunctions 1
     Array.Parallel.init (versions.Length) <| fun i ->
         let curr = aggregatedResultByVersion.[i]
-        {| versionName = versions.[i]
-           functions = curr.functions
-           enumCases = curr.enumCases |}
-
+        { version = versions.[i] |> string
+          functions = curr.functions |> Set
+          enumCases = curr.enumCases |> Set } : RawOpenGLSpecificationDetails
 
 [<EntryPoint>]
 let main argv =
     printfn "Hello World from F//!"
-    let path = @"../../../gl.xml"
+
+    System.Runtime.GCSettings.LatencyMode <- System.Runtime.GCLatencyMode.SustainedLowLatency
```

Comment:
I don't think that is necessary for this kind of Library :) (just to squeeze a few nanoseconds out of Stopwatch)

Reply:
Let us have fun with this